src/mbgl/layout/symbol_layout.cpp

```diff
@@ -229,7 +232,8 @@ void SymbolLayout::prepare(uintptr_t tileUID,
     const bool textAlongLine = layout.get<TextRotationAlignment>() == AlignmentType::Map &&
         layout.get<SymbolPlacement>() == SymbolPlacementType::Line;
 
-    for (const auto& feature : features) {
+    for (auto it = features.begin(); it != features.end(); ++it) {
+        auto feature = *it;
```

Comment:
Alas, this change only seems to take us down by 51K in the Memory.Footprint test: 6.88349e+07 => 6.87836e+07.